packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1157,13 +1159,22 @@ module.exports = function (log, config, bounces) {
   };
 
   Mailer.prototype.passwordResetEmail = function (message) {
+    log.trace('mailer.passwordResetEmail', {
+      email: message.email,
+      uid: message.uid,
+    });
```

Comment:
I think we don't need this because of https://github.com/mozilla/fxa/blob/7c53d02f24d216aaf1db4055fdf7d982d2e1f5f2/packages/fxa-auth-server/lib/senders/email.js#L404-L407